apps/formbricks-com/app/docs/self-hosting/external-auth-providers/page.mdx

```diff
@@ -0,0 +1,62 @@
+export const meta = {
+  title: "Integrating Google OAuth with Formbricks",
+  description: "Enable Google OAuth for your Formbricks instance. Follow our guide for a seamless user login experience.",
+};
+
+## Google OAuth Authentication
+
+Integrating Google OAuth with your Formbricks instance allows users to log in using their Google credentials, ensuring a secure and streamlined user experience. This guide will walk you through the process of setting up Google OAuth for your Formbricks instance.
+
+### Requirements
+
+- A Google Cloud Platform (GCP) account.
+- A Formbricks instance running and accessible.
+
+### Steps
+
+1. **Create a GCP Project**:
+   - Navigate to the [GCP Console](https://console.cloud.google.com/).
+   - From the projects list, select a project or create a new one.
+
+2. **Setting up OAuth 2.0**:
+   - If the **APIs & services** page isn't already open, open the console left side menu and select **APIs & services**.
+   - On the left, click **Credentials**.
+   - Click **New Credentials**, then select **OAuth client ID**.
+
+3. **Configure OAuth Consent Screen**:
+   - If this is your first time creating a client ID, configure your consent screen by clicking **Consent Screen**.
+   - Fill in the necessary details and under **Authorized domains**, add the domain where your Formbricks instance is hosted.
+
+4. **Create OAuth 2.0 Client IDs**:
+   - Select the appropriate application type for your project and enter any additional information required.
```

Comment:
We need to select `Web Application` for our use case 😊

<img width="612" alt="Screenshot 2023-10-25 at 10 49 08 AM" src="https://github.com/formbricks/formbricks/assets/67850763/436bc383-b372-4f9d-b05d-cbd309f0f195">